server/src/api/context.go

```diff
@@ -43,15 +43,34 @@ func (s *Server) BoardCandidateContext(next http.Handler) http.Handler {
 func (s *Server) BoardParticipantContext(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		log := logger.FromRequest(r)
-
 		boardParam := chi.URLParam(r, "id")
+
 		board, err := uuid.Parse(boardParam)
 		if err != nil {
 			common.Throw(w, r, common.BadRequestError(errors.New("invalid board id")))
 			return
 		}
 
-		user := r.Context().Value("User").(uuid.UUID)
+		userValue := r.Context().Value("User")
+		if userValue == nil {
+			log.Error("User value not found in the context")
+			common.Throw(w, r, common.BadRequestError(errors.New("user not found in the context")))
+			return
+		}
```

Comment:
I don't think it's possible for the userValue to be nil since the preceding AuthContext is checking it

Reply:
True, AuthContext is actually the methode that writes the user id into the context